### DISCUSSION TITLE
I don’t see fairings under the payload tab in the part list

### DISCUSSION THREAD

**Discussion by Davidinc2008** [likeproblem/RestockRecolour#6] [General]
Is it under another tab or is it not on the list?

**Comment by likeproblem** [comment]
Wdym

**Comment by likeproblem** [comment]
@Davidinc2008 did you mean Part List doesn't include fairings? @UncleMateo should put up updated list soon, hopefully with fairings

**Reply by Davidinc2008** [reply]
Ok, thanks!
